### DISCUSSION TITLE
HELP: Disable RAG entirely

### DISCUSSION THREAD

**Discussion by olibols** [open-webui/open-webui#5147] [General]
ow can I disable RAG when using repo pack?

I'm uploading chunks of code using repo pack. The system is explicitly designed so the LLM needs to see the entire document. However, RAG seems to be interfering by retrieving random portions of the code, even though the entire file fits easily inside the context window. This is resulting in the default Claude UI working far better than openwebui in my current use case.

This is causing issues with the intended functionality. Is there a way to disable RAG in this scenario or ensure the entire document is processed as a whole? Any guidance would be greatly appreciated.

**Comment by tobwen** [comment]
Yeah, I'm having exactly the same issue. I think you need this: https://github.com/open-webui/pipelines/blob/main/examples/pipelines/providers/anthropic_manifold_pipeline.py

**Reply by olibols** [reply]
I'm currently using this. Even while using this, I can see the little citation showing the bits of text it selected from my document and its affected by RAG settings
